net/seafile-server/Makefile

```diff
@@ -72,7 +72,9 @@ endif
 
 PKG_BUILD_DEPENDS:=vala/host libevhtp
 
-TARGET_CFLAGS += -minterlink-mips16
+ifdef CONFIG_USE_MIPS16
+	TARGET_CFLAGS += -minterlink-mips16
+endif
```

Comment:
The more appropriate fix would be not appending that flag here as it is supposed to be enabled by default by the build system.  See `include/package.mk` for details.